src/components/GroupStats.vue

```diff
@@ -3,14 +3,15 @@
     <q-card-section class="q-pb-none q-pt-sm">
       <div class="text-overline text-onsurface-m">
         <q-icon :name="icon" size="xs" color="icon-dark" />
+        <!-- eslint-disable vue-i18n/no-dynamic-keys -->
         {{ $t(title) }}
```

Comment:
```suggestion
        {{ title }}
```
Això és un bug. Aquí no hi va la $t(), ja que el que instancia el component ja ens ha de passar la propietat "title" traduïda. El linter ens l'ha fet veure!